src/settings/CommentRequiredSettings.css

```diff
@@ -0,0 +1,3 @@
+.fullWWidth {
```

Comment:
Typo. Note this isn't actually being applied because `css.fullWidth` is being requested in the JS. So if it looks correct without this class, maybe check if it actually needs to exist?